README_LINUX.md

```diff
@@ -45,6 +45,7 @@ Build requirements (optional features)
    * http://qt-project.org
    * cross-platform graphical user interface library, for Qt IDE and sclang's Qt GUI kit
    * Qt >= 5.0 should work, but build-test is done against 5.3
+   * As of this writing, Qt >= 5.6 will NOT work.
```

Comment:
[qtwebkit appears to be a dead project.](https://trac.webkit.org/wiki/BuildingQtOnLinux) but i'll try anyway...